web/pages/tour.html

```diff
@@ -119,6 +119,7 @@ <h3>Timeline</h3>
         <li>Change the interval to "Days" and explore the imagery for that month.</li>
       </ul>
     </div>
+    <p class="wv-tour-page-count">(page 1 of 6)</p>
```

Comment:
How were these `<p>` tags being added before?

Reply:
The plugin was a complete hack before. These were added into the core plugin before. There is no native support for this type of functionality.